### PR TITLE
add batch execute

### PR DESCRIPTION
**Pull request by aaronschi**

### CODE REVIEW

**Comment by mbeisel** [comment]
Looks generally good.

However,
1. Extend execute tests to contain batch executions.
2. IBMQ QPUs have a maximum number of circuits that can be executed in one job. For example, the aer_qasm_simulator has a maximum of 300 circuits. To bypass this issue, the IBMQ JobManager has been introduced (https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.managed.IBMQJobManager.html#qiskit.providers.ibmq.managed.IBMQJobManager).
--> It would be great if the qiskit-service can receive, e.g., 10,000 circuits and internally does the splitting etc., without the user needing to care about the maximum circuit limit (also add a test for a similar use-case).

**Comment by salmma** [comment]
should also solve #36

File: app/routes.py [hunk 1]
```diff
@@ -153,16 +153,20 @@ def execute_circuit():
...
     if type(impl_data) is str:
         impl_data = [impl_data]
+    
+
     transpiled_qasm = request.json.get('transpiled-qasm')
     if type(transpiled_qasm) is str:
         transpiled_qasm = [transpiled_qasm]
     bearer_token = request.json.get("bearer-token", "")
     input_params = request.json.get('input-params', "")
+    optimization_level = request.json.get('circuit transpile optimization level', 3)
```
Comment (mbeisel) [inline app/routes.py:169]: Similar to other attributes use dashes "-" in between words.
Also, maybe slightly shorten the attribute name? For example: transpilation-optimization-level

**Review by salmma — CHANGES_REQUESTED** [review]
see comments

File: app/tasks.py [hunk 2]
```diff
@@ -44,40 +46,45 @@ def execute(impl_url, impl_data, impl_language, transpiled_qasm, input_params, t
...
+    job_manager = IBMQJobManager()
+    job_result = job_manager.run(transpiled_circuit, backend=backend, name='foo')
+    # job_result = ibmq_handler.execute_job(transpiled_circuit, shots, backend)
     if job_result:
         result = Result.query.get(job.get_id())
-        result.result = json.dumps(job_result['counts'])
+        result_counts = []
+        for i, circ in enumerate(transpiled_circuit):
+            result_counts.append(job_result.results().get_counts(i))
+        result.result = json.dumps(result_counts)
```
Comment (salmma) [inline app/tasks.py:87]: if only a single circuit was executed then return the first element in the array and not the array itself like it was before this PR.

**Review by salmma — CHANGES_REQUESTED** [review]
Please extend the README.md with one example to execute multiple circuits

**Review by salmma — APPROVED** [review]
LGTM

**Pull request merged**